pepper-apis/dsm-core/src/main/resources/liquibase/CMI/PEPPER-139_view_seq_order_status.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="PEPPER-139_view_seq_order_status" author="pegah">
+        <insert tableName="access_role">
```

Comment:
Is granting existing users view_seq_order_status role, a separate Data change ticket ? 
I mean inserting rows into access_user_role_group for the new role !